package.json

```diff
@@ -1,12 +1,18 @@
 {
-  "name": "eslint-plugin-deprecation",
-  "version": "0.0.0-development",
+  "name": "@delagen/eslint-plugin-deprecation",
```

Comment:
Looks like some of the changes include in this file might not have been intended for the upstream update?

Reply:
This is because of fork. I updated this project in master branch

Reply:
@timgates42 if you has permission to merge and publish npm, I can rebase this Mr

Reply:
truncated into #39